dvc/system.py

```diff
@@ -17,7 +17,20 @@ def is_unix():
     @staticmethod
     def copy(src, dest):
         src, dest = fspath(src), fspath(dest)
-        return shutil.copyfile(src, dest)
+        system = platform.system()
+        if system == "Windows":
+            import speedcopy
+
+            return speedcopy.copyfile(src, dest)
+        else:
+            import shutil
+            import sys
+
+            if sys.version < (3, 8):
```

Comment:
```suggestion
            if sys.version_info < (3, 8):
```

Reply:
So 3.8 already uses `sendfile`? neat!

Reply:
Done.

Reply:
Upstream `speedcopy` works for both Linux and Windows now.  I will let you make the judgement as to whether to use `speedcopy` or `pyfastcopy` for the Linux side.

As you say, 3.8 should be faster, but they have taken a different approach on Windows (not using the win32 API).  I don't know whether whether that is faster/more robust, probably worth revisiting when DVC is 3.8-compatible.  It might be worth moving the version check higher up so that on 3.8 shutil always gets used.